### PR TITLE
Bug/signout issue 964

### LINKED ISSUE #964: sign-out issue on staging related to turbo that throws a CORS error (Access-Control-Allow-Origin).
Fix whatever is causing the signout redirect on staging not to work

### PR DESCRIPTION
**Pull request by mattbaumann1**
closes #964

### CODE REVIEW

File: app/views/layouts/_header.html.haml [hunk 1]
```diff
@@ -24,4 +24,4 @@
     .top-bar-right
       %ul.dropdown.menu{"data-dropdown-menu" => ""}
         = render 'layouts/header_help_links'
-        = render partial: 'devise/menu/login_items', locals: { user_is_signed_in: user_is_signed_in }
+        = render partial: 'devise/menu/login_items', locals: { user_is_signed_in: user_is_signed_in }, data: { turbo: false }
```
Comment (jseraf) [inline app/views/layouts/_header.html.haml:27]: should turbo false be here?

Comment (mattbaumann1) [reply app/views/layouts/_header.html.haml:27]: I wasn't sure about that. I don't think it is necessary. I was hoping that would work, but I can definitely take that out.

File: app/controllers/saml_sessions_controller.rb [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # require "ruby-saml"
 
 class SamlSessionsController < Devise::SamlSessionsController
+  skip_before_action :verify_authenticity_token, raise: false
   after_action :set_saml_session_index_on_session, only: :create
-  prepend_before_action :store_info_for_sp_initiated_logout, only: :destroy
+  prepend_before_action :verify_signed_out_user, :store_info_for_sp_initiated_logout, only: :destroy
```
Comment (jseraf) [inline app/controllers/saml_sessions_controller.rb:6]: Not sure this this line needs to be here. Looks like it's already in the parent class:
https://github.com/apokalipto/devise_saml_authenticatable/blob/617e7904963f68d214436244cbe0adf1d9c307f0/app/controllers/devise/saml_sessions_controller.rb

Comment (mattbaumann1) [reply app/controllers/saml_sessions_controller.rb:6]: I agree, but what I have found in the past is that for some reason the callbacks are not picked up from the parent controller and I needed to duplicate them in our overwritten file, so I did that here. You can see that with the deleted line:
prepend_before_action :store_info_for_sp_initiated_logout, only: :destroy

This comes from the parent class and was required in the previous version. As the parent file changed, I changed this one.

**Review by jseraf — APPROVED** [review]
See comments and resolve as you see fit...

**Pull request merged**